pandas/core/ops.py

```diff
@@ -168,6 +169,9 @@ def add_special_arithmetic_methods(cls, arith_method=None,
         f(op, name, str_rep, default_axis=None, fill_zeros=None, **eval_kwargs)
     comp_method : function, optional,
         factory for rich comparison - signature: f(op, name, str_rep)
+    divmod_method : function, (optional)
```

Comment:
why do we need a separate method here? cannot this be piggy-packed on arithmetic?